mcmahon.py

```diff
@@ -131,8 +146,8 @@ def pairing_score(self, player_list):
         score = 0
         temp_list = list(player_list)
         while temp_list:
-            score += abs(self.players[temp_list.pop()].mm_score -
-                         self.players[temp_list.pop()].mm_score)
+            score += abs(self.players[temp_list.pop()].mm_score[0] -
```

Comment:
I don't think this is good. Ideally we would have a class that defines not just ordering but also the difference between two scores.